steps/compliance/mrn-over-month-late/MRNOverOneMonthLate.js

```diff
@@ -52,11 +51,14 @@ class MRNOverOneMonthLate extends SaveToDraftStore {
     const benefitType = get(this, 'journey.req.session.BenefitType.benefitType');
 
     const isDWPOfficeESA = () => useDWPOfficeESA.indexOf(benefitType) !== -1;
-    const allowUC = config.get('features.allowUC.enabled') === 'true';
+    const allowUC = isFeatureFlagEnabled('allowUC');
```

Comment:
As UC benefit switched on now, can we not just remove this?

Reply:
Done